package.json

```diff
@@ -141,17 +159,29 @@
       "optional": true
     }
   },
+  "// prettierx release scripts notes": [
+    "* prepublishOnly is changed to just log some info",
+    "* prepare-release script is normally not run on prettierx,",
+    "  use prepare-extra-release instead",
+    "* build script is normally not run on prettierx,",
+    "  use build-extra-dist instead",
+    "* keeping build-extra-dist & test:dist scripts to",
+    "  help restore browser build support someday ref:",
+    "  https://github.com/brodybits/prettierx/issues/95",
+    "<!-- -->"
+  ],
   "scripts": {
-    "prepublishOnly": "echo \"Error: must publish from dist/\" && exit 1",
-    "prepare-release": "yarn && yarn build && yarn test:dist",
+    "prepublishOnly": "echo 'now publishing prettierx from project root'",
+    "prepare-release": "echo 'use prepare-extra-release for prettierx' && exit 1",
+    "prepare-extra-release": "yarn && yarn build-extra-dist && yarn test:dist",
```

Comment:
raise issues to:

- [ ] rename "extra-release" / "extra-dist" to something like "test build"
- [ ] revisit these updates in general